src/js/components/homepage/features/CovidFeature.jsx

```diff
@@ -0,0 +1,186 @@
+/*
+* CovidFeature.jsx
```

Comment:
We need to make this a container if we're connecting it to redux.

Reply:
So lets just rename this

`CovidFeatureContainer` and move it to the containers directory with the other homepage covid stuff.